src/components/todo-item-card-add/TodoItemCardAdd.tsx

```diff
@@ -1,19 +1,18 @@
 import React, { useState } from "react";
 import DatePicker from "react-datepicker";
-import { Button } from "../Button";
-import { ErrorMessage } from "../ErrorMessage/ErrorMessage";
-import { Category, Color, ICard } from "../Card";
+import { ErrorMessage } from "../error-message/ErrorMessage";
+import { Category, Color, ITodoItemCard } from "../todo-item-card";
 import "react-datepicker/dist/react-datepicker.css";
-import "./CardAddModal.css";
-import { useCardAdd } from "../../hooks/cardAdd";
-import { Loader } from "../Loader";
+import "./TodoItemCardAdd.css";
+import { useTodoItemCardAdd } from "../../hooks/useTodoItemCardAdd";
+import { Loader } from "../loader";
 
-interface CardAddModalProps {
-  onCardCreate: (card: ICard) => void;
+interface TodoItemCardAddProps {
+  onCardCreate: (card: ITodoItemCard) => void;
   onCloseModal: () => void;
 }
 
-const card: ICard = {
+const todoItemCard: ITodoItemCard = {
```

Comment:
I would rename it to the `newTodoItem` or `defaulTodoItem` or `emptyTodoItem`, cause you are using it for creating a new TodoItem. No make sense to have a `Card` postfix in the naming also, cause it's just an object.